library/Dsl/Ast/Field.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Imbo\MetadataSearch\Dsl\Ast;
+
+use Imbo\MetadataSearch\Interfaces\DslAstInterface as AstNode
+  , Imbo\MetadataSearch\Interfaces\DslAstComparisonInterface as AstComparison;
+
+/**
+ * A criteria on a fields value. Represents the name of the field and a
+ * comparison against a value.
+ */
+class Field implements AstNode {
+    /**
+     * @var string The name of the field the criteria is on
+     */
+    private $field;
+
+    /**
+     * @var Imbo\MetadataSearch\Interfaces\DslAstComparisonInterface
+     */
+    private $comparison;
+
+    /**
+     * Construct a new Field criteria.
+     * @param string $field Name of the field for the criteria
```

Comment:
For increased readability (and to conform with the docblock style in Imbo) a newline should be added between the description and first param/return declaration.